src/versions.rs

```diff
@@ -42,6 +42,84 @@ pub struct Crate {
     pub publish: Option<bool>,
 }
 
+pub async fn get_orml_crates_and_version(
+    base_url: &str,
+    version: &str,
+) -> Result<Option<(Vec<String>, String)>, Box<dyn std::error::Error>> {
+    if get_release_branches_versions(Repository::Orml)
+        .await?
+        .contains(&version.to_string())
+    {
+        let version_url = format!(
+            "{}/open-web3-stack/open-runtime-module-library/polkadot-v{}/Cargo.dev.toml",
+            base_url, version
+        );
+        let response = reqwest::Client::new()
+            .get(&version_url)
+            .header("User-Agent", "reqwest")
+            .header("Accept", "application/vnd.github.v3+json")
+            .send()
+            .await?;
+
+        let content = response.text().await?;
+
+        Ok(Some(parse_orml_workspace_members(&content)))
+    } else {
+        log::error!(
+            "No matching ORML release version found for corresponding polkadot-sdk version."
+        );
+        Ok(None)
+    }
+}
+
+fn parse_orml_workspace_members(toml_content: &str) -> (Vec<String>, String) {
+    let mut members = Vec::new();
+    let mut crates_version = String::new();
+    let mut in_workspace_members = false;
+
+    for line in toml_content.lines() {
+        if line.trim() == "[workspace]" {
+            in_workspace_members = true;
+            continue;
+        }
+
+        if line.trim().starts_with("# crates-version = \"") {
```

Comment:
That is the format how ORML crates version is fetched. The version of the crates is mentioned in comment. I am fetching it from there since all crates for a particular release version has the same version